### DISCUSSION TITLE
Geomap Initial View Coordinates

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#48857] [Dashboards, Panels & Visualizations]
Can we make the geomaps initial view, dynamic so that the map is initially centered according to the data points location.

![image](https://user-images.githubusercontent.com/43188411/167410232-6f8eef58-ddb8-4f4c-9ba3-de8ca8a7bed0.png)

I think, users can create a specific query that generates the center location (lat, lon). This query can be stored as dashboard variable and that variable can be called in the text box above image.

or you can add a query option for this specific case and store the query inside the geomap panel.

**Comment by sesoko838** [comment]
![image](https://user-images.githubusercontent.com/106267858/171265316-7f03227d-57f3-4342-8395-390a895fe62a.png)
This seems to be implemented in Grafana v 8.5.3 and works for me when querying data from SQL or InfluxDB

**Reply by DGoogol** [reply]
but this is not useful, or would it be better to have a view of all the points when it opens and, for example, every time one of the points is selected from a table, it is adjusted to that point with a dynamic zoom as well. This would avoid filtering in the queries and also redrawing the filtered map, because it would only put the focus on 1 of the points. If the idea is to see it static, it makes no sense to have zoom...

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
